Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -43,59 +43,83 @@ You can simply select _DefaultMixedRealityToolkitConfigurationProfile_ and then
 The only difference is that the *'Gaze Input'* capability is unfortunately not yet supported by Unity under 'Player Settings -> Publishing Settings -> Capabilities'. 
-To use Eye Tracking on your HoloLens 2 device, you need to manually edit the package manifest that is part of your build Visual Studio project.
-Follow these steps:
+To use eye tracking on your HoloLens 2 device, you need to manually edit the package manifest that is part of your built Visual Studio project.
 
+Follow these steps:
 1. Build your Unity project as you would normally do for _HoloLens 2_.
 2. Open your compiled Visual Studio project and then open the _'Package.appxmanifest'_ in your solution.
-3. Make sure to tick the _'Gaze Input'_ checkbox under _Capabilities_.
+3. Make sure to tick the _'GazeInput'_ checkbox under _Capabilities_.
 
 ![Enabling Gaze Input in Visual Studio](../Images/EyeTracking/mrtk_et_gazeinput.jpg)
 
-**IMPORTANT:** Don't forget to calibrate HoloLens for the current user. Without calibration, eye tracking won't work. Calibration can be found under system settings.  
+You don't see a _'GazeInput'_ capability? 
+   - Check that your system meets the [prerequisites for using MRTK](../GettingStartedWithTheMRTK.md) (in particular the Windows SDK version). 
+   - You can also manually add the entry by opening the appxmanifest in an XML editor and adding the following:
+
+```xml
+  <Capabilities>
+    <DeviceCapability Name="gazeInput" />
+  </Capabilities>
+```
+
+#### Have you eye calibrated?
+Finally, please don't forget to run through the eye calibration on your HoloLens 2. 
+The eye tracking system will not return any input if the user is not calibrated. 
+Easiest way to get to the calibration is by flipping up the visor and back down. 
+A system notification should appear welcoming you as a new user and asking you to go through the eye calibration.
+Alternatively you can find the eye calibration in the system settings: Settings -> System -> Utilities -> Open Calibration.
+
+#### Do you see the permission prompt?
+When starting the app on your HoloLens 2 for the first time, a prompt should pop up asking the user for permission to use eye tracking. 
+If it is not showing up, then that is usually an indication that the _'GazeInput'_ capability was not set. 
+
+<br>
 
-If everything is set up correctly, a prompt should pop up asking the user for permission to use Eye Tracking when you start your Unity app on a HoloLens 2 device for the first time.
+This should get you started with using eye tracking in your MRTK Unity app. 
+Don't forget to check out [our MRTK eye tracking tutorials and samples](EyeTracking_ExamplesOverview.md) demonstrating how to use eye tracking input and conveniently providing scripts that you can reuse in your projects.
 
 ---
-[Back to "Eye Tracking in the MixedRealityToolkit"](EyeTracking_Main.md)
+[Back to "Eye tracking in the MixedRealityToolkit"](EyeTracking_Main.md)
```

Comment:
Nit: I didn't catch this in a previous review, but I feeeeeeel like the back button in the browser is probably enough for people to get back and that this isn't realllllly necessary

Reply:
Clarification: Super nit

Reply:
Yeah, maybe not super necessary, but I also don't like to end up in a dead end when reaching the end of a page.